content/docs/configuration/configuration.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Configuration
+category: Configuration
+slug: 6.5.configuration
+hash: "#configuration"
+fullpath: /docs
+---
+
+Audiobookshelf is configured via environment variables.
+You can pass them to your Docker container using `-e VARIABLE=VALUE`
+or set them in `/etc/default/audiobookshelf` if you install audiobookshelf via packages.
+
+Here is a list of all available options:
+
+## Filesystem
+
+- `CONFIG_PATH` (default: `./config`)
+  - Path to the config directory.
+  - It will contain the database (users/books/libraries/settings).
+- `METADATA_PATH` (default: `./metadata`)
+  - Path to the metadata directory.
+  - It will contain cache, streams, covers, downloads, backups and logs.
+
+## External Tools
+
+- `FFMPEG_PATH` (default: `ffmpeg`)
+  - Path to the `ffmpeg` binary.
+  - If no path is set, Audiobookshelf will assume the binary to exist in the system path.
+- `FFPROBE_PATH` (default: `ffprobe`)
+  - Path to the `ffprobe` binary.
+  - If no path is set, Audiobookshelf will assume the binary to exist in the system path.
+- `TONE_PATH` (default: `tone`)
+  - Path to the `tone` binary.
+  - If no path is set, Audiobookshelf will assume the binary to exist in the system path.
```

Comment:
Do you think it is worth mentioning that these are already being installed for the Debian package?

This is not relevant for Docker either like most of these configs. These external tools seem more relevant to developers now.

Reply:
This is similar to what I wrote about `NODE_ENV` above but here I can attest that I actually dug though the code when I first tried out Audiobookshelf since I wanted it to use my system FFmpeg and not specify a specific version. Having this documented would have been nice back then. Although, there was a bug in this back then :D